src/type.ts

```diff
@@ -0,0 +1,56 @@
+export interface Product {
+    id: number;
+    title: string;
+    description: string;
+    price: number;
+    images: [];
+    creationAt: string;
+    updatedAt: string;
+    category: {};
+}
+
+export interface ProductId {
+    id: number;
+    title: string;
+    price: number;
+    description: string;
+    images: [];
+    creationAt: string;
+    updatedAt: string;
+    category: {};
+}
+
+export interface Category {
+    id: number;
+    name: string;
+    image: string;
+    creationAt: string;
+    updatedAt: string;
+}
+
+export interface CategoryId {
```

Comment:
You can reuse your types so you don't need to duplicate "export interface Category" by adding "export interface CategoryId"

Reply:
fixed